wls-ergebnismeldung-service/src/main/resources/db/migrations/h2/V2_0__createTableWahlscheine.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE Wahlscheine
+(
+    wahlID                           VARCHAR(1024) NOT NULL,
+    wahlbezirkID                     VARCHAR(1024) NOT NULL,
+    stimmabgabevermerke              BIGINT       NOT NULL,
```

Comment:
```suggestion
    stimmabgabevermerke              BIGINT        NOT NULL,
```

Wobei bei mir die Datei beim Speichern kompakter ist ....

![grafik](https://github.com/user-attachments/assets/c8a92e19-a7cd-4a65-a43f-2321c5d32e9e)